modules/simple_forms_api/app/services/simple_forms_api/confirmation_email.rb

```diff
@@ -40,6 +41,10 @@ def send
 
     def form_specific_data
       email, first_name = case @form_number
+                          when 'vba_21_0972'
+                            return unless Flipper.enabled?(:form21_0972_confirmation_email)
+
+                            [@form_data['claimant_email'], @form_data.dig('claimant_full_name', 'first')]
```

Comment:
Should this be referencing the `preparer` instead of the `claimant`?

Reply:
I noticed `@form_data` is not made public via exclusion from the list of `attr_reader` attributes. 

I understand this is completely allowed and valid, I'm curious though about the design decision and when to include an attribute as being included with `attr_reader`.

Reply:
Typically we'd want it included in the `attr_reader` when external controllers/methods are directly accessing it.